### DISCUSSION TITLE
Protocoll: GenericFileException; Exception thrown: OCP\Files\GenericFileException

### DISCUSSION THREAD

**Discussion by Rimos911** [nextcloud/all-in-one#4507] [Questions] [answered]
Hello,

can someone tell me what this error message means and how I can fix it.
My NC is up-to-date (28.0.4); Password-App: 2024.4.20
This error occurs even after uninstalling the app

**Protocoll**

Level: Fatal
Anwendug: passwords
Message:

GenericFileException
Exception thrown: OCP\Files\GenericFileException
Exception thrown: OCP\Files\GenericFileException

GenericFileException
file_put_contents failed

Thanks in advanced

**Comment by szaimen** [comment, marked as answer]
Hi, this needs to be reported to and fixed in the passwords app
